client/gutenberg/extensions/map-block/save.js

```diff
@@ -0,0 +1,46 @@
+/** @format */
+
+/**
+ * External dependencies
+ */
+
+import { Component } from '@wordpress/element';
+import classnames from 'classnames';
+
+/**
+ * Internal dependencies
+ */
+
+import { settings } from './settings.js';
+
+class MapSave extends Component {
+	render() {
+		const { className, attributes } = this.props;
+		const { map_style, points, zoom, map_center, marker_color, align } = attributes;
+		const atavistAlignClass = value => {
+			switch ( value ) {
+				case 'left':
+				case 'right':
+				case 'center':
+				case 'full':
+					return 'atavist-block-align-' + value;
```

Comment:
Can we rename `atavist-block-align-` to `jetpack-block-align-` ?

Reply:
Yes, will do. I will need the `atavist-`* alignment classes in order for this block to play nicely with Atavist themes, but I believe I'll be able to accomplish this using Gutenberg's extensibility hooks, so I'll put these back in through a plugin.

Reply:
Got rid of all of the atavist class names in https://github.com/Automattic/wp-calypso/pull/27836/commits/be1654f671aa25df8f79a332b3dda5f2ff4c7169.